api/src/main/java/org/apache/iceberg/types/TypeUtil.java

```diff
@@ -355,6 +355,17 @@ public static Schema reassignOrRefreshIds(
     return new Schema(struct.fields(), refreshIdentifierFields(struct, schema));
   }
 
+  /**
+   * Assigns fresh ids from the {@link GetID getId function} for all fields in a type.
+   *
+   * @param type a type
+   * @param getId an id assignment function
+   * @return an structurally identical type with new ids assigned by the nextId function
```

Comment:
Name here is incorrect, nextId is not defined, getId is

Reply:
Thanks, fixed.